tests/integration/server/api/users/[id]/index.get.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it } from "vitest"
+import { setup, $fetch } from "@nuxt/test-utils/e2e"
+import { v4 as uuidv4 } from "uuid"
+import prisma from "../../../../../../server/utils/prisma"
+
+describe("Api Route /api/users", async () => {
+  await setup()
+
+  it ("Should get a user", async () => {
```

Comment:
maybe name it should get a user by id and then add another test to check if it works with the username as well